exercises/simple-linked-list/src/test/scala/SimpleLinkedListTest.scala

```diff
@@ -0,0 +1,56 @@
+import org.scalatest.{FlatSpec, Matchers}
+
+class SimpleLinkedListTest extends FlatSpec with Matchers {
```

Comment:
I know it may be not easy for this exercise, but maybe one can agree on the tests in `x-common/exercises/simple-linked-list`?

Perhaps there are also some more useful tests in the [Haskell TestSuite](https://github.com/exercism/xhaskell/blob/master/exercises/simple-linked-list/test/Tests.hs)?